docker/scripts/load-backup-archives

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+DUMP_DIR="${MUSICBRAINZ_SEARCH_DUMP_DIR:-/var/cache/musicbrainz/solr-backups}"
+SOLR_BACKUPS_DIR="${SOLR_BACKUPS_DIR:-/var/solr/data/backups}"
+SOLR_BASE_URL="http://${SOLR_LOCAL_HOST:-localhost}:${SOLR_PORT:-8983}"
+
+SCRIPT_NAME=$(basename "$0")
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [<options>]
+
+Load MusicBrainz Solr collections from fetched Solr backup archives.
+
+Options:
+  -h, --help    Print this help message
+
+Environment:
+  MUSICBRAINZ_SEARCH_DUMP_DIR   path to directory with Solr backup archives
+                                (default: /var/cache/musicbrainz/solr-backups)
+  SOLR_BACKUPS_DIR              path to directory with Solr backup files
+                                (default: /var/solr/data/backups)
```

Comment:
Just so I understand correctly, is the difference between these directories just that `SOLR_BACKUPS_DIR` contains the extracted files from `MUSICBRAINZ_SEARCH_DUMP_DIR`?

That would makes sense. I was just slightly confused after `load-backup-archives` logged this:

> Delete the no-longer-needed backup 'annotation'...

I thought perhaps this meant I didn't need to run `remove-backup-archives`, but they're for different backups.

Reply:
> Just so I understand correctly, is the difference between these directories just that `SOLR_BACKUPS_DIR` contains the extracted files from `MUSICBRAINZ_SEARCH_DUMP_DIR`?

Yes.

> That would makes sense. I was just slightly confused after `load-backup-archives` logged this:
> 
> > Delete the no-longer-needed backup 'annotation'...

It removes the SolrCloud collection _backup_ `annotation`.
This _backup_ is stored under the directory `/var/solr/data/backups/annotation`.

> I thought perhaps this meant I didn't need to run `remove-backup-archives`, but they're for different backups.

_Backup archives_ are just the tar archives `*.tar.zst` of these backups.

So you can run `remove-backup-archives` whenever you are happy with your setup and want to save disk space.

Reply:
Thanks for the clarification!

Reply:
Further clarified the wording through the commit https://github.com/metabrainz/mb-solr/pull/61/commits/cd6d832e84f010a94c370626aa8d1fd923661cbd.